src/wasm-lib/kcl/src/executor.rs

```diff
@@ -1980,9 +1980,52 @@ impl From<crate::settings::types::ModelingSettings> for ExecutorSettings {
     }
 }
 
+/// Create a new zoo api client.
+#[cfg(not(target_arch = "wasm32"))]
+pub fn new_zoo_client(token: Option<String>, engine_addr: Option<String>) -> Result<kittycad::Client> {
+    let user_agent = concat!(env!("CARGO_PKG_NAME"), ".rs/", env!("CARGO_PKG_VERSION"),);
+    let http_client = reqwest::Client::builder()
+        .user_agent(user_agent)
+        // For file conversions we need this to be long.
+        .timeout(std::time::Duration::from_secs(600))
+        .connect_timeout(std::time::Duration::from_secs(60));
+    let ws_client = reqwest::Client::builder()
+        .user_agent(user_agent)
+        // For file conversions we need this to be long.
+        .timeout(std::time::Duration::from_secs(600))
```

Comment:
Can we at least make it adjustable via an env var or similar?

Reply:
someone else can do it in another PR, and then it should match kittycad.rs and we need them there too

Reply:
lets jsut make this PR about the variables that are already in every sdk

Reply:
if we add more here we need them in EVERY OTHER SDk no more one offs

Reply:
like if you wanna really go deep just make a raw dog client and pass it thru, thats literally an option these are just helpers

Reply:
yea yea it's all good heh, you convinced me at "they can raw dog it" :stuck_out_tongue: